tests/test_deserialization.py

```diff
@@ -826,6 +826,32 @@ class AliasingUserSerializer(Schema):
         assert result['email'] == 'foo@bar.com'
         assert result['age'] == 42
 
+    def test_deserialize_with_attribute_param_error_returns_field_name_not_attribute_name(self):
+        class AliasingUserSerializer(Schema):
+            username = fields.Email(attribute='email')
+            years = fields.Integer(attribute='age')
+        data = {
+            'username': 'foobar.com',
+            'years': '42'
+        }
+        result, errors = AliasingUserSerializer().load(data)
+        assert errors
+        assert errors['username'] == [u'"foobar.com" is not a valid email address.']
+
+    def test_deserialize_with_attribute_param_error_returns_load_from_not_attribute_name(self):
+        class AliasingUserSerializer(Schema):
+            name = fields.String(load_from='Name')
+            username = fields.Email(attribute='email', load_from='UserName')
+            years = fields.Integer(attribute='age', load_from='Years')
+        data = {
+            'Name': 'Mick',
+            'UserName': 'foobar.com',
+            'years': 'abc'
+        }
+        result, errors = AliasingUserSerializer().load(data)
+        assert errors['UserName'] == [u'"foobar.com" is not a valid email address.']
```

Comment:
I'm not sure about this behavior: why should the key on errors change if the input value is missing vs. present? My expectation (as a user) would be that the key on errors is always the same as the name of the field.